packages/cli/src/commonlib/logger.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is this a renaming of *cli/src/commonlib/**log.ts***?

Reply:
logger.ts is a refactored log provider which will replace log.ts in the future

Reply:
So are you going to do the refactor and own the new logger? If so, feel free to change the CODEOWNERS then no need to wait unnecessary reviewers each time~